### PR TITLE
Travis fix

### PR DESCRIPTION
**Pull request by jaeilepp**
Updated the scipy version and fixed some PEP8 violations that had sneaked through.
Let's see if travis passes now.

### CODE REVIEW

**Comment by dengemann** [comment]
@jaeilepp it seems you're T3000 -- from the future right?

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ env:
     #
     # Conda currently has packaging bug with mayavi/traits/numpy where 1.10 can't be used
     # but breaks sklearn on install; hopefully eventually the NUMPY=1.9 on 2.7 full can be removed
-    - PYTHON=2.7 DEPS=full TEST_LOCATION=src NUMPY="=1.9" SCIPY="=0.16"
+    - PYTHON=2.7 DEPS=full TEST_LOCATION=src NUMPY="=1.9" SCIPY="=0.17"
```
Comment (Eric89GXL) [inline .travis.yml]: IIRC this doesn't work with the numpy, scipy, and mayavi config. Did you test with a venv locally?

**Comment by Eric89GXL** [comment]
Travis appears to have missed the memo about this PR, I'll try closing and reopening

File: .travis.yml (hunk 1 above)
Comment (jaeilepp) [inline .travis.yml]: I couldn't replicate the failures on my virtualenv, but a travis build on my fork seems to pass.

Comment (Eric89GXL) [inline .travis.yml]: Cool, hopefully they updated things to make it work

**Comment by jaeilepp** [comment]
It's green!

**Comment by Eric89GXL** [comment]
Thanks @jaeilepp!

**Comment by agramfort** [comment]
Great !

**Pull request merged**